### PR TITLE
fix: updates SimpleTable look and feel to match original implementation

### LINKED ISSUE #783: SimpleTable: styling should match influxdata/ui
The styling for `SimpleTable` should match the styling found for the same visualization in influxdata/ui

Mainly, update the styling to match the clockface 3 updates for influxdata/ui. Various subcomponents need to be updated including `Table` and `DapperScrollbars`

### PR DESCRIPTION
**Pull request by TCL735**
Closes #783  
Closes https://github.com/influxdata/ui/issues/5239

- Fixes the styling of SimpleTable to match the original implementation
- Re-ports the subcomponents of SimpleTable
- organizes the scss variables into a file
- fixes the exports of supporting types to allow importing just a single file rather than multiple files with different paths
- takes the latest fixes from Clockface for Icons and related code

https://user-images.githubusercontent.com/10736577/181662732-a400e527-49d9-4998-936d-1fa3f62de1f7.mp4

### CODE REVIEW

File: giraffe/src/components/SimpleTable/PagedTable.tsx [hunk 1]
```diff
@@ -231,8 +233,30 @@ const PagedTable: FC<Props> = ({result, properties}) => {
     setTotalPages,
   } = useContext(PaginationContext)
   const [height, setHeight] = useState(0)
+  const [headerHeight, setHeaderHeight] = useState(0)
+  const [rowHeight, setRowHeight] = useState(0)
   const ref = useRef()
 
+  if (headerHeight === 0) {
+    const calculatedHeaderHeight =
+      document.querySelector<HTMLElement>(`#${pagedTableHeaderId}`)
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PagedTable.tsx:242]: I already mentioned using the querySelector vs a ref in the other PR, so I won't repeat it.

File: giraffe/src/components/SimpleTable/PaginationNav/Button/Button.tsx [hunk 2]
```diff
@@ -6,34 +6,88 @@ import {ButtonBase, ButtonBaseRef} from './ButtonBase'
...
+      onMouseLeave,
+      icon = '',
+      active = false,
+      testID = 'button',
+      type = ButtonType.Button,
+      size = ComponentSize.Small,
+      placeIconAfterText = false,
+      shape = ButtonShape.Default,
+      color = ComponentColor.Default,
+      status = ComponentStatus.Default,
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/Button/Button.tsx:53]: alphabetize these?

File: giraffe/src/components/SimpleTable/PaginationNav/Button/Button.tsx [hunk 3]
```diff
@@ -6,34 +6,88 @@ import {ButtonBase, ButtonBaseRef} from './ButtonBase'
...
-        tabIndex={0}
-        ref={ref}
+        onMouseOut={onMouseOut}
+        onMouseOver={onMouseOver}
+        onMouseEnter={onMouseEnter}
+        onMouseLeave={onMouseLeave}
+        className={className}
+        titleText={titleText || text}
+        disabledTitleText={disabledTitleText}
+        tabIndex={!!tabIndex ? tabIndex : 0}
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/Button/Button.tsx:81]: if there's no organization scheme, alphabetization is always welcomed!

File: giraffe/src/components/SimpleTable/PaginationNav/PaginationDirectionItem.tsx [hunk 4]
```diff
@@ -22,18 +31,43 @@ export const PaginationDirectionItem = forwardRef<
...
+      size = ComponentSize.Medium,
+      isActive,
+    },
     ref
   ) => {
+    const paginationClassName = classnames('cf-pagination--item--container', {
+      [`${className}`]: className,
+    })
+      .split(' ')
+      .reduce((accum, current) => styleReducer(styles, accum, current), '')
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/PaginationDirectionItem.tsx:50]: doesn't `classnames` do the same thing the `styleReducer` does?

File: giraffe/src/components/SimpleTable/PaginationNav/PaginationInput.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  ComponentSize,
+  IconFont,
+  InputType,
+  StandardFunctionProps,
+} from '../../../types'
+
+export interface PaginationInputProps extends StandardFunctionProps {
+  currentPage: number
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void | undefined
+  onClick?: (e?: MouseEvent<HTMLButtonElement>) => void
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/PaginationInput.tsx:23]:
```suggestion
  onChange?: (event: ChangeEvent<HTMLInputElement>) => void | undefined
  onClick?: (event?: MouseEvent<HTMLButtonElement>) => void
```

File: giraffe/src/components/SimpleTable/PagedTable.tsx [hunk 6]
```diff
@@ -299,7 +323,16 @@ const PagedTable: FC<Props> = ({result, properties}) => {
 
   const inner =
     !!size &&
-    tables.map((t, tIdx) => <InnerTable table={t} key={`table${tIdx}`} />)
+    tables.map((t, tIdx) => (
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PagedTable.tsx:326]:
```suggestion
    tables.map((table, index) => (
```

File: giraffe/src/components/SimpleTable/PaginationNav/PaginationNav.tsx [hunk 7]
```diff
@@ -113,15 +151,56 @@ export const Pagination = forwardRef<PaginationNavRef, PaginationNavProps>(
 
       if (page !== activePage && notOutOfBound) {
         setActivePage(page)
+        setInputPage(page)
         onChange(page)
       }
     }
 
+    const onInputChange = (e: ChangeEvent<HTMLInputElement>) => {
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/PaginationNav.tsx:159]:
```suggestion
    const onInputChange = (event: ChangeEvent<HTMLInputElement>) => {
```

File: giraffe/src/components/SimpleTable/PaginationNav/PaginationNav.tsx [hunk 8]
```diff
@@ -113,15 +151,56 @@ export const Pagination = forwardRef<PaginationNavRef, PaginationNavProps>(
...
+      let parsedValue = parseInt(e.target.value, 10)
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/PaginationNav.tsx:160]:
```suggestion
      let parsedValue = parseInt(event.target.value, 10)
```

File: giraffe/src/components/SimpleTable/PaginationNav/PaginationTruncationItem.tsx [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+  ComponentStatus,
+  StandardFunctionProps,
+} from '../../../types'
+
+// Styles
+import styles from './Pagination.scss'
+import {styleReducer} from '../../../utils/styleReducer'
+
+export interface PaginationTruncationItemProps extends StandardFunctionProps {
+  onClick?: (e?: MouseEvent<HTMLElement>) => void
```
Comment (hoorayimhelping) [inline giraffe/src/components/SimpleTable/PaginationNav/PaginationTruncationItem.tsx:22]:
```suggestion
  onClick?: (event?: MouseEvent<HTMLElement>) => void
```

**Review by hoorayimhelping — APPROVED** [review]
a couple of minor suggestions

File: giraffe/src/components/SimpleTable/PaginationNav/PaginationDirectionItem.tsx (hunk 4 above)
Comment (TCL735) [reply giraffe/src/components/SimpleTable/PaginationNav/PaginationDirectionItem.tsx:50]: No. The `styleReducer` changes the string to a unique string, while `classnames` still has the readable, original strings concatenated together. What the `styleReducer` is doing is actually reaching into the `styles` object that is imported and getting the unqiue string back.

File: giraffe/src/components/DapperScrollbars.tsx [hunk 10]
```diff
@@ -76,12 +79,13 @@ export const DapperScrollbars: FunctionComponent<DapperScrollbarsProps> = ({
   noScrollY = false,
   autoSizeWidth = false,
   autoSizeHeight = false,
-  thumbStopColor = InfluxColors.Pool,
-  thumbStartColor = InfluxColors.Star,
+  thumbStopColor = 'rgba(255, 255, 255, 0.25)',
+  thumbStartColor = 'rgba(255, 255, 255, 0.25)',
```
[the remark this answers is not in the document]
Comment (TCL735) [reply giraffe/src/components/DapperScrollbars.tsx:83]: Same color, no error.

File: giraffe/src/components/SimpleTable/InnerTable.tsx [hunk 11]
```diff
@@ -1,19 +1,28 @@
...
+  pagedTableIds: {pagedTableHeaderId, pagedTableBodyId},
+  table,
+}) => {
   const headers = Object.values(table.cols).map(c => {
     if (c.name === 'table') {
       return (
         <Table.HeaderCell
           key="htable"
           className={`${styles['cf-table--header-cell']}`}
+          // verticalAlignment={VerticalAlignment.Top}
```
[the remark this answers is not in the document]
Comment (TCL735) [reply giraffe/src/components/SimpleTable/InnerTable.tsx]: Decided to use it instead.

**Pull request merged**